as2_utilities/as2_external_object_to_tf/CMakeLists.txt

```diff
@@ -0,0 +1,115 @@
+cmake_minimum_required(VERSION 3.5)
+set(PROJECT_NAME as2_external_object_to_tf)
+project(${PROJECT_NAME})
+set(as2_external_object_to_tf_FOUND 1)
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 17)
+endif()
+
+# set Release as default
+if(NOT CMAKE_BUILD_TYPE)
+  set(CMAKE_BUILD_TYPE Release)
+endif()
+
+find_package(ament_cmake REQUIRED)
+find_package(rosidl_default_generators REQUIRED)
```

Comment:
Why not add this to PROJECT_DEPENDENCIES?

Reply:
done